File: backend/Persistence/Models/Shop.cs
using System;
using System.Collections.Generic;

namespace Persistence.Models;

public partial class Shop
{
    public Guid Id { get; set; }

    public bool DisplayWindowAvailable { get; set; }

    public double StorageRoomSize { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime UpdatedAt { get; set; }
}

File: mobile/lib/src/features/dashboard/domain/entities/shop.dart
import 'package:equatable/equatable.dart';

class Shop extends Equatable {
  final bool displayWindowAvailable;
  final double storageRoomSize;

  const Shop({
    required this.displayWindowAvailable,
    required this.storageRoomSize,
  });

  @override
  List<Object?> get props => [displayWindowAvailable, storageRoomSize];
}
